Puc/v4p2/Plugin/UpdateChecker.php

```diff
@@ -399,6 +400,49 @@ public function addCheckForUpdatesLink($pluginMeta, $pluginFile) {
 		}
 
 		/**
+		 * Add a "View Details" link to the plugin row in the "Plugins" page. By default,
+		 * the new link will appear after the "Visit plugin site" link.
+		 *
+		 * You can change the link text by using the "puc_view_details_link-$slug" filter.
+		 * Returning an empty string from the filter will disable the link.
+		 *
+		 * @param array $pluginMeta Array of meta links.
+		 * @param string $pluginFile
+		 * @return array
+		 */
+		public function addViewDetailsLink($pluginMeta, $pluginFile) {
```

Comment:
As mentioned above, I recommend adding another argument so that you can get the plugin header:
```php
addViewDetailsLink($pluginMeta, $pluginFile, $pluginData = array())
```